### PR TITLE
Configure pre-commit hooks

### PR DESCRIPTION
**Pull request by Mmesek**
Adds Black to format code on commit and configures isort to use black's profile for sorting imports

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -22,6 +22,13 @@ repos:
     rev: 5.10.1
     hooks:
       - id: isort
+        args: ["--profile", "black", "--filter-files"]
+
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        args: ["-l", "119"]
```
Comment (Ksenofanex) [inline .pre-commit-config.yaml:32]: I think we can add Python's version after this line, as suggested in the documentation. Check [black's docs](https://black.readthedocs.io/en/stable/integrations/source_version_control.html).

`language_version: python3.10`

**Comment by Ksenofanex** [comment]
I suggest to add `- id: check-json` to pre-commit's hook, just in case.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (Mmesek) [reply .pre-commit-config.yaml:32]: Wouldn't it be better to add global language version to the configuration?
[Pre-commit docs](https://pre-commit.com/#top_level-default_language_version)
```yaml
default_language_version:
    python: python3.10
```
Also, wouldn't it also be better to make it a separate PR? 👀 Not sure if it's in-scope of this one

Comment (Ksenofanex) [reply .pre-commit-config.yaml:32]: Either is fine, the choice is yours. :)

About the separate PR, no need. We're already reviewing the pre-commit hooks, starting with black. You can change the title of the PR if you want, but not necessary IMO.

Comment (Mmesek) [reply .pre-commit-config.yaml:32]: Done. #Resolved?

**Review by Ksenofanex — APPROVED** [review]

**Review by MilaDog — APPROVED** [review]

**Pull request merged**